core/Cargo.toml

```diff
@@ -5,7 +5,7 @@ name = "deno"
 version = "0.18.0"
 edition = "2018"
 description = "A secure JavaScript/TypeScript runtime built with V8, Rust, and Tokio"
-authors = ["The deno authors <bertbelder@nodejs.org>"]
```

Comment:
Does crates.io accept it without an e-mail address?
Maybe we can create a shared mailbox, e.g. info@deno.land.

Reply:
It does accept it without an email address: https://crates.io/crates/deno_cli

Reply:
Cool, thanks